code/modules/cargo/packs.dm

```diff
@@ -41,6 +41,9 @@
 		for(var/item in contains)
 			new item(C)
 
+/datum/supply_pack/proc/get_access()
+	return access
```

Comment:
Why are you making a proc that returns a variable when you could just access the variable?

Reply:
That.. is a good point. I was expecting to do thing differently when I started, I changed it up now.